### PR TITLE
Scrubbing azure env reference and cleaning up environments content

### LINKED ISSUE #173: scrub docs for "Azure environment" language
with removal of `rad env init azure`, we should check that we don't use that verbiage in our docs anymore

### PR DESCRIPTION
**Pull request by Reshrahim**
1. Removed references to Azure environments
    - excludes webapp, dapr, eshop and container apps samples as they are being worked on by individual PM owners
    - excludes faq.md as it changed in #171
 2. Cleanup of environments concept and managing environments
     - Completed and cleaned up creating an environment under the environments concepts page
     -  Excludes environments concerns cleanup as Aaron is working on refining it 
     - Moved Managing environments to administrator guide and cleaned up the steps

### CODE REVIEW

File: docs/content/concepts/environments-concept/_index.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ title: "Environments concept"
 linkTitle: "Environments"
 description: "Learn about Radius Environments"
 weight: 300
+slug: environment
```
Comment (AaronCrawfis) [inline docs/content/concepts/environments-concept/_index.md:7]: 🔥🔥🔥
Great use of slugs!

File: docs/content/getting-started/tutorial/_index.md [hunk 2]
```diff
@@ -9,42 +9,77 @@ no_list: true
 
 ## Overview
 
-This tutorial will teach you how to deploy a website as a Radius application from first principles. You will learn:  
+This tutorial will teach you how to deploy a website as a Radius application from first principles. You will take away the following
```
Comment (AaronCrawfis) [inline docs/content/getting-started/tutorial/_index.md]: Did you want to update tutorial content here, or capture it as part of #177 ?

Comment (Reshrahim) [reply docs/content/getting-started/tutorial/_index.md]: Ooh my bad this has tutorial changes as well :(. I think the tutorial update can be captured via #177

**Review by AaronCrawfis — DISMISSED** [review]
Looks great! For the tutorial changes that were folded in there are a couple options:

1. Leave them in, merge the edge branch into your tutorial branch, and continue developing the tutorial there.
2. Undo the tutorial changes manually in this branch to leave just the environments updates
3. Create a fresh branch off of edge, paste your environments updates, and open a new PR

Option 1 is probably easiest

File: docs/content/getting-started/tutorial/_index.md (hunk 2 above)
Comment (Reshrahim) [reply docs/content/getting-started/tutorial/_index.md]: Removed the tutorial content from this branch. @AaronCrawfis - can you approve the PR ?

**Review by AaronCrawfis — APPROVED** [review]

**Comment by AaronCrawfis** [comment]
:shipit:

**Pull request merged**